static/js/copy-code-button.js

```diff
@@ -1,12 +1,13 @@
-/* global clipboard */
-/* eslint-disable no-console */
-
 function addCopyButtons(clipboard) {
-  const button = document.getElementById('copy-code-button');
-
   document.querySelectorAll('pre > code').forEach(function (codeBlock) {
+    var button = document.createElement('button');
+    button.className =
+      'copy-code-button absolute text-white text-xs right-[3.5rem] bg-red-500 p-1 hover:text-red-700 [&.active]:bg-green-600 mt-[0.75rem] mr-[0.75rem]';
+    button.type = 'button';
+    button.innerText = 'Copy';
+
     button.addEventListener('click', function () {
-      clipboard.writeText(codeBlock.textContent).then(
+      clipboard.writeText(codeBlock.innerText).then(
```

Comment:
Could you explain the decision for replacing `textContent` with `innerText`? I'm just curious.

Reply:
```
- textContent is not aware of styles. So it returns the entire content
- innerText is aware of CSS.
```